build.gradle

```diff
@@ -243,18 +243,18 @@ dependencies {
     exclude group: 'junit', module: 'junit'
     exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
   }
-  testImplementation group: 'com.azure', name: 'azure-core-test', version: '1.1.0', {
+  testImplementation group: 'com.azure', name: 'azure-core-test', version: '1.2.1', {
```

Comment:
i don't think we are using it at all ;)

Reply:
we are using. but as I see we can get rid of. we used in mock.

Reply:
so yea - we are not using it :D

Reply:
we are using a mock object from the lib.